src/snowcli/snow_connector.py

```diff
@@ -16,8 +17,16 @@
 from snowcli.config import cli_config, get_default_connection
 from snowcli.exception import SnowflakeConnectionError, InvalidConnectionConfiguration
 
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.serialization import Encoding
+from cryptography.hazmat.primitives.serialization import NoEncryption
+from cryptography.hazmat.primitives.serialization import PrivateFormat
+from cryptography.hazmat.primitives.serialization import load_pem_private_key
```

Comment:
Please move the imports to `load_pem_to_der` so we don't import those modules every time